packages/edit-site/src/components/sync-state-with-url/use-sync-canvas-mode-with-url.js

```diff
@@ -59,7 +59,7 @@ export default function useSyncCanvasModeWithURL() {
 	useEffect( () => {
 		currentCanvasInUrl.current = canvasInUrl;
 		if (
-			canvasInUrl === undefined &&
+			( canvasInUrl === undefined || canvasInUrl === 'view' ) &&
```

Comment:
The canvas mode can only be `view`, `edit` or `undefined`, correct? If so, could this be simplified to `canvasInUrl !== 'edit'`?

Reply:
Good idea!